File: src/main/java/upbrella/be/user/dto/token/KakaoOauthInfo.java
package upbrella.be.user.dto.token;

import lombok.*;
import org.springframework.beans.factory.annotation.Value;
import org.springframework.stereotype.Component;

@Getter
@Builder
@Component
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
public class KakaoOauthInfo {

    @Value("${KAKAO_CLIENT_ID_DEV}")
    private String clientId;
    @Value("${KAKAO_CLIENT_SECRET_DEV}")
    private String clientSecret;
    @Value("${KAKAO_REDIRECT_URI_DEV}")
    private String redirectUri;
    @Value("${KAKAO_LOGIN_URI_DEV}")
    private String loginUri;
}

File: src/main/kotlin/upbrella/be/user/dto/token/KakaoOauthInfo.kt
package upbrella.be.user.dto.token

import org.springframework.beans.factory.annotation.Value
import org.springframework.stereotype.Component

@Component
class KakaoOauthInfo(
    @Value("\${KAKAO_CLIENT_ID_DEV}")
    val clientId: String,

    @Value("\${KAKAO_CLIENT_SECRET_DEV}")
    val clientSecret: String,

    @Value("\${KAKAO_REDIRECT_URI_DEV}")
    val redirectUri: String,

    @Value("\${KAKAO_LOGIN_URI_DEV}")
    val loginUri: String
)
